### PR TITLE
fastd 17 (new formula)

### PR DESCRIPTION
**Pull request by MoritzFago**

### CODE REVIEW

File: Library/Formula/fastd.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
+class Fastd < Formula
+  desc "Fast and Secure Tunnelling Daemon"
+  homepage "https://projects.universe-factory.net/projects/fastd"
+  url "https://projects.universe-factory.net/attachments/download/81/fastd-17.tar.xz"
+  sha256 "26d4a8bf2f8cc52872f836f6dba55f3b759f8c723699b4e4decaa9340d3e5a2d"
+  head "git://git.universe-factory.net/fastd/"
+  depends_on "cmake" => :build
```
Comment (bfontaine) [inline Library/Formula/fastd.rb]: Please leave an empty line before the first `depends_on`; thanks!

File: Library/Formula/fastd.rb [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  depends_on "libsodium"
+  depends_on "bison" => :build # fastd requires bison >= 2.5
+  depends_on "pkg-config" => :build
+  depends_on "json-c"
+  depends_on "openssl" => :optional
+
+  def install
+    mkdir "fastd-build" do
+      if build.with? "openssl"
+        system "cmake", "-DENABLE_LTO=ON -DENABLE_OPENSSL=ON", *std_cmake_args, ".."
```
Comment (mistydemeo) [inline Library/Formula/fastd.rb]: The splatted argument needs to come at the end of the argument list.

**Comment by MoritzFago** [comment]
I forgot the tun/tap dependency and I'll close this now and make a new pr this evening.

**Comment by MoritzFago** [comment]
I forgot the tun/tap dependency and I'll close this now and make a new pr this evening.

**Comment by bfontaine** [comment]
@MoritzFago You can add commits to a PR to update it :wink:

**Comment by MoritzFago** [comment]
@bfontaine yes thats true, but because i though i had to rebase it, i opened a new PR so the current PR is #48217. Maybe you could take a look at it?

**Pull request closed without merging**